src/signals/incident-management/containers/Dashboard/components/AreaChart/AreaChart.tsx

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { useEffect, useState } from 'react'
+
+import { useDispatch } from 'react-redux'
+import { Link } from 'react-router-dom'
+import vegaEmbed from 'vega-embed'
+import type { EmbedOptions } from 'vega-embed'
+
+import LoadingIndicator from 'components/LoadingIndicator'
+import { showGlobalNotification } from 'containers/App/actions'
+import { VARIANT_ERROR, TYPE_LOCAL } from 'containers/Notification/constants'
+
+import { ComparisonRate } from './ComparisonRate'
+import { AreaChartWrapper as Wrapper } from './styled'
+import type { ComparisonRateType } from './types'
+import { formatData, getMaxDomain, getToday, getPercentage } from './utils'
+import { INCIDENTS_URL } from '../../../../routes'
+import { constants, getAreaChartSpec } from '../../charts'
+import type { AreaChartValue as Value } from '../../charts'
+import { useGetAreaChart } from '../../hooks/useGetAreaChart'
+import { ModuleTitle } from '../ModuleTitle'
+
+const embedOptions: EmbedOptions = {
+  actions: false,
+  timeFormatLocale: constants.timeFormatLocale,
+  mode: 'vega-lite',
+}
+
+interface Props {
+  queryString: string
+}
+
+export const AreaChart = ({ queryString }: Props) => {
+  const [data, setData] = useState<Value[]>()
```

Comment:
IMO
I might just put AreaChartValue here. It's more descriptive. 
If you would use something like namespace.Value at least a understand where Value derives from.
And it's not like Props what you expect by default.

Reply:
I feel you.